projects/packages/forms/src/dashboard/index.js

```diff
@@ -47,8 +48,10 @@ window.addEventListener( 'load', () => {
 	const root = createRoot( container );
 	root.render(
 		<>
-			<RouterProvider router={ router } />
-			<DashboardNotices />
+			<ThemeProvider>
+				<RouterProvider router={ router } />
+				<DashboardNotices />
+			</ThemeProvider>
```

Comment:
Out of curiosity, why is the ThemeProvider needed here?

Reply:
Because of the `spacing-base` CSS variable. That variable is not strictly necessary, it just makes the spacing decision (multiple of 8px) explicit due to a standard.